eventuate-local-java-cdc-connector-common/src/main/java/io/eventuate/local/common/BinlogEntry.java

```diff
@@ -2,7 +2,23 @@
 
 import io.eventuate.common.eventuate.local.BinlogFileOffset;
 
+import java.nio.charset.StandardCharsets;
+
 public interface BinlogEntry {
   Object getColumn(String name);
   BinlogFileOffset getBinlogFileOffset();
+
+  default String convertColumnToString(String name) {
```

Comment:
I think a better name would be `getStringColumn()`

Reply:
I will change